migrate/schema.sql

```diff
@@ -238,23 +238,23 @@ DROP TABLE IF EXISTS `community-voices_oberlin_metadata`;
 /*!40101 SET character_set_client = utf8 */;
 CREATE TABLE `community-voices_oberlin_metadata` (
   `id` int(21) NOT NULL AUTO_INCREMENT,
-  `source_type` varchar(100) DEFAULT NULL,
-  `interviewee_or_source_document` varchar(100) DEFAULT NULL,
-  `organization` varchar(100) DEFAULT NULL,
-  `sponsor_organization` varchar(100) DEFAULT NULL,
-  `topic` varchar(100) DEFAULT NULL,
-  `interviewee_email` varchar(100) DEFAULT NULL,
-  `interviewee_phone` varchar(100) DEFAULT NULL,
-  `url_consent_interview` varchar(100) DEFAULT NULL,
-  `t1_survey` varchar(100) DEFAULT NULL,
-  `t2_survey` varchar(100) DEFAULT NULL,
-  `url_transcription` varchar(100) DEFAULT NULL,
-  `url_article` varchar(100) DEFAULT NULL,
-  `date_article_approved` varchar(100) DEFAULT NULL,
-  `url_photograph` varchar(100) DEFAULT NULL,
-  `suggested_photo_source` varchar(100) DEFAULT NULL,
-  `suggested_photo_in_cv` varchar(100) DEFAULT NULL,
-  `create_a_slide` tinyint(1) NOT NULL,
+  `source_type` varchar(255) DEFAULT NULL,
+  `interviewee_or_source_document` varchar(255) DEFAULT NULL,
+  `organization` varchar(255) DEFAULT NULL,
+  `sponsor_organization` varchar(255) DEFAULT NULL,
+  `topic` varchar(255) DEFAULT NULL,
+  `interviewee_email` varchar(255) DEFAULT NULL,
+  `interviewee_phone` varchar(255) DEFAULT NULL,
+  `url_consent_interview` varchar(255) DEFAULT NULL,
+  `t1_survey` varchar(255) DEFAULT NULL,
+  `t2_survey` varchar(255) DEFAULT NULL,
+  `url_transcription` varchar(255) DEFAULT NULL,
+  `url_article` varchar(255) DEFAULT NULL,
+  `date_article_approved` varchar(255) DEFAULT NULL,
+  `url_photograph` varchar(255) DEFAULT NULL,
+  `suggested_photo_source` varchar(255) DEFAULT NULL,
+  `suggested_photo_in_cv` varchar(255) DEFAULT NULL,
+  `create_a_slide` varchar(255) DEFAULT NULL,
```

Comment:
@Sammidysam Why did you change the type of the "create_a_slide" field? I think that Rowan and John want to use this field in the future to determine whether to pair with a quote or not, so a boolean datatype (tinyint(1)) seems more appropriate to me.

Reply:
This is totally my bad and thanks for finding this.  I was not looking at the field types closely enough.

Reply:
Fixed